posthog/api/cohort.py

```diff
@@ -1,21 +1,35 @@
 import csv
+from datetime import datetime
 from typing import Any, Dict, List, Optional, cast
 
+from django.conf import settings
 from django.db.models import Count, QuerySet
+from django.db.models.expressions import F
+from django.utils import timezone
 from rest_framework import serializers, viewsets
 from rest_framework.permissions import IsAuthenticated
 from rest_framework.request import Request
 from sentry_sdk.api import capture_exception
 
+from ee.clickhouse.client import sync_execute
```

Comment:
A bit late to the party, but I'm assuming during this migration we're okay with `posthog-foss` being broken?

Reply:
yessir